.github/workflows/docker.build.yml

```diff
@@ -56,23 +56,36 @@ jobs:
           tags: ${{ steps.docker_meta.outputs.tags }}
           labels: ${{ steps.docker_meta.outputs.labels }}
       #Staging: Create pull request on deploy repo from a merged PR
+      - name: Setup python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+      - run: pip install -r .github/workflows/requirements.txt
       -
         name: create pull request for deploy on staging
         uses: actions/checkout@v3
-        # remove this if statement after fixing the PR
-        if: 1==2
         with:
           repository: wmde/wbaas-deploy
       # Make changes to pull request here
-      - # TODO: Remove this if statement after fixing the PR
-        if: 1==2
-        run: |
-          get_tag=$(echo ${{ steps.docker_meta.outputs.tags }} | cut -d':' -f2)
-          yq -i '.image.tag=$get_tag' k8s/helmfile/env/local/ui.values.yaml.gotmpl
-          yq -i '.image.tag=$get_tag' k8s/helmfile/env/staging/ui.values.yaml.gotmpl
+      - run:
+          |
+          python - <<EOF
+          from ruamel.yaml import YAML
+          import sys
+
+          yaml=YAML()
+          with open("k8s/helmfile/env/local/ui.values.yaml.gotmpl") as doc:
+              data=yaml.load(doc)
+          data['image']['tag']="sha-"+${{env.GITHUB_SHA}}[:7]
```

Comment:
looks like perhaps this `${{env.GITHUB_SHA}}[:7]` isn't quite doing it; I wonder if that's because this isn't being correctly parsed or instead if this environment variable is just empty

Reply:
The variable is empty. It has not value `on pull_request` since this branch has no merge commits but has one after a push and it is the latest commit in main. see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

On push see: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push